pkg/asset/machines/machineconfig/ipv6.go

```diff
@@ -36,7 +36,7 @@ func ForDualStackAddresses(role string) (*mcfgv1.MachineConfig, error) {
 		},
 		Spec: mcfgv1.MachineConfigSpec{
 			Config:          rawExt,
-			KernelArguments: []string{"ip=dhcp,dhcp6"},
+			KernelArguments: []string{"br-ex:ip=dhcp,dhcp6"},
```

Comment:
@cybertron, your experience will be needed here. To me it feels that we do actually need `ip=dhcp,dhcp6` on the interface before we create `br-ex`. Or not?

Reply:
Yes, this won't work if configure-ovs is managing br-ex. We need the underlying interface to get an address first.

Reply:
@cybertron @mkowalski Indeed, you're right. I specified the name of the first interface.
I'm not sure if the approach of specifying the name of interface on configuration is widely used. Thoughts?